templates/submit.sh.epp

```diff
@@ -14,6 +14,8 @@
 #SBATCH --partition=<%= $slurm_partitions %>
 <% } %>
 
+<%# default behaviour if no additions are provided -%>
+<% if $additions == '' { %>
```

Comment:
I would prefer we set a default value in common.yaml instead of checking if `$additions` is empty.

Reply:
sure thing! I've done that in my latest commit.